hw/ip/i2c/dv/cov/i2c_cov_excl.el

```diff
@@ -0,0 +1,72 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Transition from host mode states to AcquireStart and Idle
+// wont happen since `start_det` and `stop_det` used to
+// detect start and stop conditions are not asserted in
+// host mode.
+// Refer OT issue https://github.com/lowRISC/opentitan/issues/18651
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: jdonjdon
+// Format Version: 2
+// Date: Tue May 23 22:40:14 2023
+// ExclMode: default
+//==================================================
```

Comment:
Please add annotation like this
https://github.com/lowRISC/opentitan/blob/c6d13942ec3df7be63ff6ab172f59823d3e9be3c/hw/ip/keymgr/dv/cov/keymgr_cov_excl.el#L5

Reply:
Added annotation in [aaa5c0a](https://github.com/lowRISC/opentitan/compare/4af57a4c4b2049986260996cdabe3bd5ce0829c4..aaa5c0a1964d4310465c5a9a97fc55a4e65a23aa)

Reply:
awesome! thanks.